alibi/explainers/partial_dependence.py

```diff
@@ -388,18 +350,31 @@ def check_feature(f):
             else:
                 check_feature(f)
 
-    def _params_sanity_checks(self,
-                              estimator: Union[BaseEstimator, 'PDEstimatorWrapper'],
-                              response_method: Literal['auto', 'predict_proba', 'decision_function'] = 'auto',
-                              method: Literal['auto', 'recursion', 'brute'] = 'auto',
-                              kind: Literal['average', 'individual', 'both'] = 'average'
-                              ) -> Tuple[str, str, str]:
+    @staticmethod
+    def _blackbox_params_sanity_checks(method: Literal['auto', 'recursion', 'brute'] = 'auto') -> None:
         """
-        Parameters sanity checks. Most of the code is borrowed from:
+        Parameters sanity checks for black-box models.
+
+        Parameters
+        ----------
+        method
+            See :py:meth:`alibi.explainers.partial_dependence.PartialDependence.explain` method.
+        """
+        if method != Method.BRUTE:
+            raise ValueError(f"method='{method}' is invalid. For black-box models "
+                             f"the method must be {Method.BRUTE}.")
```

Comment:
Should this be `Method.BRUTE.value`?

Reply:
not necessarily. see `_sklearn_params_sanity_checks` where it works just fine.